### PR TITLE
Add container model

### PR DESCRIPTION
**Pull request by agrare**
Add a container model belonging to a container_group and store the
memory and cpu limits/requests

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -225,7 +247,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.datetime "archived_on"
-    t.bigint "tenant_id"
+    t.bigint "tenant_id", null: false
```
Comment (Ladas) [inline db/schema.rb]: hm, I didn't push these in my previous migration (we are missing the check)

File: db/schema.rb [hunk 2]
```diff
@@ -119,6 +119,28 @@
...
+    t.jsonb "resource_timestamps", default: {}
+    t.datetime "resource_timestamps_max"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.datetime "source_created_at"
+    t.datetime "source_deleted_at"
+    t.datetime "archived_on"
+    t.index ["archived_on"], name: "index_containers_on_archived_on"
+    t.index ["container_group_id", "name"], name: "index_containers_on_container_group_id_and_name", unique: true
+    t.index ["container_group_id"], name: "index_containers_on_container_group_id"
```
Comment (Ladas) [inline db/schema.rb]: I think this index might be redundant, the unique index should be used for query ```.where(:container_group_id => XY)```

File: db/schema.rb (hunk 1 above)
Comment (agrare) [reply db/schema.rb]: Yeah we need to re-run db:migrate to write out the db/schema.rb file after changing a migration

File: db/schema.rb (hunk 2 above)
Comment (agrare) [reply db/schema.rb]: :+1:

**Pull request merged**